Kickstarter-iOS/ViewModels/SettingsCurrencyPickerCellViewModel.swift

```diff
@@ -0,0 +1,46 @@
+import KsApi
+import Library
+import Prelude
+import ReactiveSwift
+import Result
+
+protocol SettingsCurrencyPickerCellViewModelInputs {
+  func configure(with cellValue: SettingsCellValue)
+  func didSelectCurrency(currency: Currency)
+}
+
+protocol SettingsCurrencyPickerCellViewModelOutputs {
+  var notifyCurrencyPickerCellRemoved: Signal<Void, NoError> { get }
+  var showCurrencyChangeAlert: Signal<Currency, NoError> { get }
+}
+
+protocol SettingsCurrencyPickerCellViewModelType {
+  var inputs: SettingsCurrencyPickerCellViewModelInputs { get }
+  var outputs: SettingsCurrencyPickerCellViewModelOutputs { get }
+}
+
+final class SettingsCurrencyPickerCellViewModel: SettingsCurrencyPickerCellViewModelOutputs,
+SettingsCurrencyPickerCellViewModelInputs, SettingsCurrencyPickerCellViewModelType {
+
+  public init() {
+    self.notifyCurrencyPickerCellRemoved = self.selectedCurrencyProperty.signal.ignoreValues()
+
+    self.showCurrencyChangeAlert = self.selectedCurrencyProperty.signal.skipNil()
```

Comment:
Can we combine these two outputs into just one? It seems that `notifyCurrencyPickerCellRemoved` is probably not needed anymore.